tests/js/spec/views/performance/landing/widgets/widgetContainer.spec.jsx

```diff
@@ -710,7 +717,7 @@ describe('Performance > Widgets > WidgetContainer', function () {
         'p50 Duration'
       );
 
-      expect(wrapper.find('HighlightNumber').text()).toEqual('51s');
+      expect(wrapper.find('HighlightNumber').text()).toEqual('534ms');
```

Comment:
Why is this assertion updated? I thought this is only renaming stuff, did some logic change?

Reply:
yes this part of the code was updated https://github.com/getsentry/sentry/blob/6f110dafc1089862facb501adbc06f3be3fcd67d/static/app/views/performance/landing/widgets/transforms/transformMetricsToArea.tsx#L87